src/lattice/lattice.js

```diff
@@ -10,6 +10,14 @@ import {LatticeVectors} from "./lattice_vectors";
 import {primitiveCell} from "../cell/primitive_cell";
 import {Cell} from "../cell/cell";
 
+/**
+ * Additive factor used to calculate the new lattice size for non-periodic systems.
+ * The ensures that a non-periodic structure will have a minimum lattice size of 2x2x2.
+ * The minimum lattice size ensures that the structures coordinates will fit inside the cell
+ * used to hold the structure.
+ */
+export const nonPeriodicLatticeMinimumSizeAngstroms = 2.0;
```

Comment:
This is clearly not Angstrom?

![](https://media1.giphy.com/media/J6AJL4BFuTHACf2Idc/giphy.gif)